src/Hit/ColorTerminal.hs

```diff
@@ -116,18 +117,22 @@ arrow = " ➤  "
 data Answer = Y | N
 
 -- | Parse an answer to 'Answer'
-yesOrNo :: Text -> Maybe Answer
-yesOrNo (T.toLower -> answer )
-    | T.null answer = Just Y
+yesOrNo :: Answer -> Text -> Maybe Answer
```

Comment:
Could you patch the documentation to this function saying that it also should take a default answer?

Reply:
Also, probably it's better to name it `yesOrNoWithDefault`?